Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -114,18 +116,18 @@ void TilePartitionerNoise::writeDebugData(const noise::ModuleGroup & central,
                                           const noise::ModuleGroup & combiner)
 {
 
-    std::cout << "Writing debug modules for " << mDebugWriter->getCornerDescription() << "...\n" ;
+    LOG_DEBUG << "Writing debug modules for " << mDebugWriter->getCornerDescription() << "...\n" ;
     mDebugWriter->writeDebugGroup(central, "central");
     mDebugWriter->writeDebugGroup(border_h, "border_h");
     mDebugWriter->writeDebugGroup(border_v, "border_v");
     mDebugWriter->writeDebugGroup(combiner, "combiner");
-    std::cout << "Debug modules written. Press 'q' <ENTER> to stop debugging.\n" <<
+    LOG_DEBUG << "Debug modules written. Press 'q' <ENTER> to stop debugging.\n" <<
         "Press <ENTER> to write the next set...\n";
```

Comment:
This (and probably the other messages in this file) should be `LOG_INFO` or whatever the user debug writer is called.